presto-cli/src/main/java/io/prestosql/cli/CsvPrinter.java

```diff
@@ -31,13 +31,38 @@
 
```

Comment:
Please update commit message. `Minor refactorings for csv output format` is very ambiguous.  Maybe something like `Use enum for CLI CSV output format`?